api/cmd/portainer/main.go

```diff
@@ -88,26 +88,6 @@ func initDataStore(flags *portainer.CLIFlags, fileService portainer.FileService,
 		return nil
 	}
 
-	// Init sets some defaults - its basically a migration
-	err = store.Init()
-	if err != nil {
-		log.Fatalf("failed initializing data store: %v", err)
-	}
-
-	if isNew {
-		// from MigrateData
-		store.VersionService.StoreDBVersion(portainer.DBVersion)
```

Comment:
Since this isNew logic gets removed.  Where does a new DB get initialised and the current portainer DBVersion being set.  I don't see all this logic elsewhere?   
Also updateSettingsFromFlags? where has that logic been moved to?